geti_sdk/deployment/deployed_model.py

```diff
@@ -326,7 +352,7 @@ def load_inference_model(
                 raise ValueError(
                     f"Tiling is not supported for domain {self._converter.domain}"
                 )
-            self._tiler = tiler_type(model=model, execution_mode="sync")
+            self._tiler = tiler_type(model=model, execution_mode="async")
```

Comment:
should it be parametrized?

Reply:
Well ideally it should always run in async mode, because it's faster. However, I have been experiencing issues with it in my tests, I think it's safer to leave it in sync mode. We can switch it over later when it's working reliably and we have a proper test for it.